app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt

```diff
@@ -1,8 +1,18 @@
 package org.p2p.wallet.common.mvp
 
 import androidx.annotation.AnimRes
+import androidx.annotation.StringRes
+import com.google.android.material.snackbar.Snackbar
 
 interface BaseFragmentContract {
     fun overrideEnterAnimation(@AnimRes animation: Int)
     fun overrideExitAnimation(@AnimRes animation: Int)
+    // new snackbar
+    fun showUiKitSnackBar(
```

Comment:
I know, strange stuff to put showUiKitSnackBar here, but it's the only way to show snackbar on MainFragment